tool/tsh/db.go

```diff
@@ -964,3 +1015,40 @@ const (
 	// dbFormatYAML prints database info as YAML.
 	dbFormatYAML = "yaml"
 )
+
+const (
+	// errDBCmdUnsupportedTLSRouting is the error message printed when some
+	// database subcommands are not supported because TLS routing is enabled.
+	errDBCmdUnsupportedTLSRouting = `"%v" is not supported when TLS routing is enabled on the Teleport Proxy Service.
+
+Please use "tsh db connect" or "tsh proxy db" to connect to the database.`
+
+	// errDBCmdUnsupportedDBProtocol is the error message printed when some
+	// database subcommands are run against unsupported database protocols.
+	errDBCmdUnsupportedDBProtocol = `"%v" is not supported for %v databases.
```

Comment:
nit: It is a bit unintuitive  that `err...` prefixed  variable name is used for string where in code seeing `errDBCmdUnsupportedDBProtocol` I will assume that the type of errDBCmdUnsupportedDBProtocol is `error`